--- a/libnvtx_common.jl	2019-10-25 16:09:36.638690884 +0200
+++ b/libnvtx_common.jl	2019-10-25 16:33:25.506013480 +0200
@@ -28,14 +28,6 @@
 
 # Skipping MacroDefinition: NVTX_RESOURCE_ATTRIB_STRUCT_SIZE ( ( uint16_t ) ( sizeof ( nvtxResourceAttributes_v0 ) ) )
 
-const nvtxMark = nvtxMarkA
-const nvtxRangeStart = nvtxRangeStartA
-const nvtxRangePush = nvtxRangePushA
-const nvtxNameCategory = nvtxNameCategoryA
-const nvtxNameOsThread = nvtxNameOsThreadA
-const nvtxDomainCreate = nvtxDomainCreateA
-const nvtxDomainRegisterString = nvtxDomainRegisterStringA
-const nvtxDomainNameCategory = nvtxDomainNameCategoryA
 const nvtxRangeId_t = UInt64
 const nvtxDomainRegistration_st = Cvoid
 const nvtxDomainRegistration = nvtxDomainRegistration_st
@@ -109,10 +109,6 @@
 const nvtxResourceHandle = Cvoid
 const nvtxResourceHandle_t = Ptr{nvtxResourceHandle}
 const NVTX_RESOURCE_CLASS_CUDA = 4
-const nvtxNameCuDevice = nvtxNameCuDeviceA
-const nvtxNameCuContext = nvtxNameCuContextA
-const nvtxNameCuStream = nvtxNameCuStreamA
-const nvtxNameCuEvent = nvtxNameCuEventA
 
 @cenum nvtxResourceCUDAType_t::UInt32 begin
     NVTX_RESOURCE_TYPE_CUDA_DEVICE = 262145
