--- a/libcusparse_common.jl	2019-09-19 14:17:40.834901970 +0200
+++ b/libcusparse_common.jl	2019-09-19 15:04:05.039885162 +0200
@@ -6,7 +6,5 @@
 const cusparseContext = Cvoid
 const cusparseHandle_t = Ptr{cusparseContext}
-const cusparseMatDescr = Cvoid
-const cusparseMatDescr_t = Ptr{cusparseMatDescr}
 const cusparseSolveAnalysisInfo = Cvoid
 const cusparseSolveAnalysisInfo_t = Ptr{cusparseSolveAnalysisInfo}
 const csrsv2Info = Cvoid
@@ -172,3 +169,25 @@
 const cusparseDnVecDescr_t = Ptr{cusparseDnVecDescr}
 const cusparseSpMatDescr_t = Ptr{cusparseSpMatDescr}
 const cusparseDnMatDescr_t = Ptr{cusparseDnMatDescr}
+
+
+"""
+Describes shape and properties of a CUSPARSE matrix. A convenience wrapper.
+
+Contains:
+* `MatrixType` - a [`cusparseMatrixType_t`](@ref)
+* `FillMode` - a [`cusparseFillMode_t`](@ref)
+* `DiagType` - a [`cusparseDiagType_t`](@ref)
+* `IndexBase` - a [`cusparseIndexBase_t`](@ref)
+"""
+struct cusparseMatDescr
+    MatrixType::cusparseMatrixType_t
+    FillMode::cusparseFillMode_t
+    DiagType::cusparseDiagType_t
+    IndexBase::cusparseIndexBase_t
+    function cusparseMatDescr(MatrixType,FillMode,DiagType,IndexBase)
+        new(MatrixType,FillMode,DiagType,IndexBase)
+    end
+end
+
+const cusparseMatDescr_t = Ptr{cusparseMatDescr}
