--- a/libnvtx_common.jl	2019-10-25 16:09:36.638690884 +0200
+++ b/libnvtx_common.jl	2019-10-25 16:25:03.653940666 +0200
@@ -5,7 +5,7 @@
 
 # Skipping MacroDefinition: NVTX_INLINE_STATIC inline static
 
-const NVTX_DECLSPEC = NVTX_INLINE_STATIC
+# Skipping MacroDefinition: NVTX_DECLSPEC
 
 # Skipping MacroDefinition: NVTX_VERSIONED_IDENTIFIER_L3 ( NAME , VERSION ) NAME ## _v ## VERSION
 # Skipping MacroDefinition: NVTX_VERSIONED_IDENTIFIER_L2 ( NAME , VERSION ) NVTX_VERSIONED_IDENTIFIER_L3 ( NAME , VERSION )
