--- a/libnvtx_common.jl	2019-10-25 16:09:36.638690884 +0200
+++ b/libnvtx_common.jl	2019-10-25 16:33:25.506013480 +0200
@@ -72,6 +64,10 @@
 end
 
 
+struct payload_t
+    ullValue::UInt64
+end
+
 struct nvtxEventAttributes_v2
     version::UInt16
     size::UInt16
@@ -96,6 +92,10 @@
 end
 
 
+struct identifier_t
+    ullValue::UInt64
+end
+
 struct nvtxResourceAttributes_v0
     version::UInt16
     size::UInt16
