--- a/libcudnn_common.jl	2019-10-23 17:52:17.651150610 +0200
+++ b/libcudnn_common.jl	2019-10-23 17:53:07.195648729 +0200
@@ -403,5 +400,8 @@
-struct cudnnAlgorithmUnionStruct
-    algo::Algorithm
-end
-
-const cudnnAlgorithm_t = cudnnAlgorithmUnionStruct
+# FIXME: can't use such a union as the type in a ccall expression
+#Algorithm = Union{cudnnConvolutionFwdAlgo_t, cudnnConvolutionBwdFilterAlgo_t, cudnnConvolutionBwdDataAlgo_t, cudnnRNNAlgo_t, cudnnCTCLossAlgo_t}
+#struct cudnnAlgorithm_t
+#    algo::Algorithm
+#end
+#
+#const cudnnAlgorithm_t = cudnnAlgorithmUnionStruct
+const cudnnAlgorithm_t = Cint
